ext/rugged/rugged_commit.c

```diff
@@ -107,9 +105,7 @@ static VALUE rb_git_commit_author_GET(VALUE self)
 	git_commit *commit;
 	Data_Get_Struct(self, git_commit, commit);
 
-	return rugged_signature_new(
-		git_commit_author(commit),
-		git_commit_message_encoding(commit));
+	return rugged_signature_new(git_commit_author(commit));
```

Comment:
I think this is probably an OK assumption to make.  I haven't seen any exceptions regarding this in production (only the path stuff)